Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandPartyJoin.java

```diff
@@ -0,0 +1,56 @@
+package com.universeprojects.miniup.server.commands;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.universeprojects.cacheddatastore.CachedDatastoreService;
+import com.universeprojects.cacheddatastore.CachedEntity;
+import com.universeprojects.miniup.server.ODPDBAccess;
+import com.universeprojects.miniup.server.commands.framework.Command;
+import com.universeprojects.miniup.server.commands.framework.UserErrorMessage;
+
+/**
+ * 
+ * 
+ * @author dylan
+ * 
+ * Allows a player to join a party if they are already not apart of a party, not in combat, the party is accepting members, or already apart of the same party,
+ *
+ */
+public class CommandPartyJoin extends Command {
+
+	public CommandPartyJoin(ODPDBAccess db, HttpServletRequest request,
+			HttpServletResponse response) {
+		super(db, request, response);
+	}
+
+	@Override
+	public void run(Map<String, String> parameters) throws UserErrorMessage {
+		ODPDBAccess db = getDB();
+		CachedDatastoreService ds = getDS();
+
+		CachedEntity character = db.getCurrentCharacter();
```

Comment:
Party code will be null when not in a party. We should throw a meaningful error message.